docs/source/contributing/developing.adoc

```diff
@@ -221,12 +221,14 @@ $ make integration
 
 NOTE: By default `make integration` only runs tests which are tagged as `@basic`.
 
-To run all the test, use the following command:
+To run all tests, use the following command:
 
 ----
 $ make integration_all
 ----
 
+NOTE: There is also a `make integration_pr` target which is being run as part of pull request testing.
```

Comment:
I think @robin-owen tries to convert all NOTE, TIP, etc into the multi-line version.  

\[NOTE]
\====
\====

Best to align.

@robin-owen, why don't you schedule a meeting and give us a short heads up on the most common style violations we use?

Reply:
@hferentschik Aah, missed that, will then help @robin-owen with this file. Update soon.

+1 for the sync around best practices and common mistakes we do.

Reply:
@agajdosi - fyi pull request #1684. I guess in the end it does not matter whether you update this here. I take it that @robin-owen needs to rebase this pull request anyways an then he can take care of it then.

> +1 for the sync around best practices and common mistakes we do.

Right, at least the core devs should be aligned on this, otherwise, there will be inconsistencies in no time. 

@robin-owen I think we need two things here. First the additional section in the writing docs document which outlines some of these syles, eg multiline NOTE over NOTE:, use of '{project}' over Minishift, etc. Once this is in place we still should have a short sync were we together go through this list once. Just to make sure everyone read it. @robin-owen WDYT?